### PR TITLE
Website Style Upgrade

### PR DESCRIPTION
**Pull request by LegendaryLinux**
## Changelog
- Themes added to AutoWorld class
- Themes added to WebHost
- Accessibility upgrades
- New fonts
- New backgrounds
- New styles

### CODE REVIEW

File: worlds/AutoWorld.py [hunk 1]
```diff
@@ -132,6 +136,10 @@ class World(metaclass=AutoWorldRegister):
     item_names: Set[str]  # set of all potential item names
     location_names: Set[str]  # set of all potential location names
 
+    # If the game displays all contained items to the user, this flag pre-fills the hint system with this information
```
Comment (Berserker66) [inline worlds/AutoWorld.py:139]: this is gone. PR should not re-add it.

File: WebHostLib/__init__.py [hunk 2]
```diff
@@ -70,6 +70,13 @@ def to_url(self, value):
 app.jinja_env.filters['suuid'] = lambda value: base64.urlsafe_b64encode(value.bytes).rstrip(b'=').decode('ascii')
 
 
+def get_world_theme(game_name: str):
+    for game, world in AutoWorldRegister.world_types.items():
```
Comment (Berserker66) [inline WebHostLib/__init__.py:74]: You can just lookup instead of iterating; `AutoWorldRegister.world_types[game_name].web.theme`

Comment (LegendaryLinux) [reply WebHostLib/__init__.py]: This one has been fixed and pushed.

File: worlds/AutoWorld.py (hunk 1 above)
Comment (LegendaryLinux) [reply worlds/AutoWorld.py:139]: Please be more specific with your feedback. What is "this"?

Comment (Berserker66) [reply worlds/AutoWorld.py:139]: The 4 lines this PR is adding at this marker point. sending_visible is no longer a thing.

Comment (LegendaryLinux) [reply worlds/AutoWorld.py]: Those lines have been removed. I'm not actually sure how they snuck back in there, considering I never touched them while working on the WebHost.

**Comment by Berserker66** [comment]
I guess last question then, squash or rebase?

**Comment by LegendaryLinux** [comment]
Just squash it, I think.

**Review by Berserker66 — APPROVED** [review]

**Comment by LegendaryLinux** [comment]
Last two commits removed references to the Z5Client from the OoT English tutorial and fixed an overflow problem on code blocks with very long lines. Should have also fixed that merge conflict.

**Review by Ijwu — APPROVED** [review]

**Comment by Ijwu** [comment]
> Just squash it, I think.

K, squashing. Great job on this, it looks amazing. Big credit to our designer, too.

**Pull request merged**